js/data.js

```diff
@@ -5,11 +5,14 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
 window.addEventListener('beforeunload', windowString);
 function windowString(event) {
   var entriesJSON = JSON.stringify(data.entries);
```

Comment:
you want to store the entire data object in local storage so that all of the data info is saved there, that way you can use the `data.view` property for keeping the view value accurate